core/client/README.md

```diff
@@ -78,4 +78,10 @@ The `Paginator` class will automatically handle API errors and stop iterating if
 ## Built-in Pagination Support
 Some APIs have built-in support for pagination, the API class exposes a method that will return the `Paginator` object for you. For example, the `EndpointAgentsApi` class has a `getEndpointAgentsPaginated` method that returns a `Paginator` object for fetching paginated endpoint agents.
```

Comment:
Probably should have raised this in the last PR tbh, but it might be worth mentioning that `Paginator` implements `Iterable` and has a built in `stream()` method, because for most users those will be the "real" interfaces that they interact with.

And actually, maybe a better example would be to show the iterator in action, rather than just the generated code (which people could find for themselves if they wanted), so something like:

```java
public class Example {
    public static void main(String[] args) {
        ApiClient client = NativeApiClient.builder().bearerToken(BEARER_TOKEN).build();

        EndpointAgentsApi endpointAgentsApi = new EndpointAgentsApi(client);
        for (EndpointAgent agent : endpointAgentsApi.getEndpointAgentsPaginated(null, AID, List.of(), false, null, null)) {
                System.out.println(agent.getName());
        }
    }
}
```

(Maybe use a simpler API in the example too 😄)

Reply:
Ah, ignore me, I see that's all covered above